backend/routes/follow_route.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+let router = express.Router();
+const { emailExists } = require("../utils/user.js");
+const User = require("../models/user.js");
+
+// User is the one who initiates the follow to target user
+router.post("/follow", async (req, res) => {
+    const {userID, target_userID} = req.body; // get the userID and target user ID from the request body, can be found using username or email
```

Comment:
very quick fix: it is best to retrieve the paramaters indivually such as with const userID=req.body.user_id or something similar, this in order to account for a case where the frontend doesn't send the parameters in the expected order (this same thing actually caused issues a few PRs ago), should probably also require a small modification to the unit tests to account for this new way of getting the parameters

also if you could add a check for when the parameters are missing in the request that would be great, along with a few tests for it

Reply:
you can reference other route files to see how parameters are retrieved and also to see how we can check if they were actually passed to the request or not